frontend/src/app/pages/about-me/profiles/profiles.component.html

```diff
@@ -0,0 +1,17 @@
+<div *ngIf="person">
+  <div class="item">
```

Comment:
use semantics tags for this. see https://www.tracylum.com/blog/2015-12-21-the-benefits-of-semantic-html/